go/bindings/combine.go

```diff
@@ -151,14 +156,22 @@ func (d *Combine) Destroy() {
 	}
 }
 
+type StatsMessage interface {
```

Comment:
nit: Use `proto.Unmarshaler` instead ?

Reply:
Thanks for sharing. I didn't know that was a thing.